From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Gunnar Beutner <gbeutner@serenityos.org>
Date: Fri, 18 Jun 2021 15:06:37 +0200
Subject: [PATCH] Replace some size_t => Bitu

FIXME: No information as to why this is a thing exists as of yet, fill
       them in when that stuff is known.
---
 src/hardware/gus.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/hardware/gus.cpp b/src/hardware/gus.cpp
index 355a082..6c20b5c 100644
--- a/src/hardware/gus.cpp
+++ b/src/hardware/gus.cpp
@@ -245,7 +245,7 @@ private:
 	void PopulatePanScalars() noexcept;
 	void PopulateVolScalars() noexcept;
 	void PrepareForPlayback() noexcept;
-	size_t ReadFromPort(const size_t port, const size_t iolen);
+	Bitu ReadFromPort(const Bitu port, const Bitu iolen);
 	void RegisterIoHandlers();
 	void Reset(uint8_t state);
 	void SetLevelCallback(const AudioFrame &level);
@@ -253,7 +253,7 @@ private:
 	void UpdateDmaAddress(uint8_t new_address);
 	void UpdateWaveMsw(int32_t &addr) const noexcept;
 	void UpdateWaveLsw(int32_t &addr) const noexcept;
-	void WriteToPort(size_t port, size_t val, size_t iolen);
+	void WriteToPort(Bitu port, Bitu val, Bitu iolen);
 	void WriteToRegister();
 
 	// Collections
